src/assets/svg/home/integration.svg

```diff
@@ -0,0 +1,9 @@
+</svg>
```

Comment:
A quick-n-dirty example of stacking multiple FontAwesome icons:

```
const DBSyncIcon = () => (
  <span className="fa-layers fa-fw">
    <FontAwesomeIcon icon={faDatabase} size="2x" />
    <FontAwesomeIcon icon={faCircle} transform="grow-10 down-10 right-16" style={{ color: '#f5f5f8' }} />
    <FontAwesomeIcon icon={faSyncAlt} transform="down-10 right-16" />
  </span>
);
```
... which produces this:

![Screen Shot 2020-02-18 at 6 16 46 PM](https://user-images.githubusercontent.com/440299/74795453-d74f2f00-527a-11ea-9150-6d948c98ae07.png)